lib/bcp47.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module Bcp47
+  require 'bcp47/parser'
+  require 'bcp47/record'
+  require 'bcp47/registry'
+end
```

Comment:
Sticking this in `lib/` since it seems ripe for extraction to a new BCP 47 gem. The current gems for BCP 47 are outdated and unmaintained, as best I can tell.